src/cf-buttons/molecules/buttons-with-icons.less

```diff
@@ -0,0 +1,48 @@
+@import '../atoms/buttons.less';
+
+//
+// Button icons
+//
+
+.a-btn_icon__left {
+    padding: 0 unit( 10.5px / @btn-font-size, em) 0 0;
+    border-right: 1px solid @btn-text; // Fallback
+    border-right: 1px solid fade(@btn-text, 40%);
+    margin: 0 unit( 7px / @btn-font-size, em) 0 0;
```

Comment:
Do we need to reset the top bottom and left values here? The base `a-btn` resets everything to zero, but I'm not sure if that's added back in somewhere.